tests/test_search.py

```diff
@@ -0,0 +1,37 @@
+from jupyddl.heuristics import goal_count_heuristic, zero_heuristic
+from jupyddl.automated_planner import AutomatedPlanner
+from jupyddl.dijkstra import DijkstraBestFirstSearch
+from jupyddl.a_star import AStarBestFirstSearch
+from jupyddl.bfs import BreadthFirstSearch
+from jupyddl.dfs import DepthFirstSearch
+from os import path
+import coloredlogs
+import sys
+
+apla = AutomatedPlanner("data/domain.pddl", "data/problem.pddl")
+
+
+def test_searchDFS():
+    dfs = DepthFirstSearch(apla)
+    _, ct = dfs.search()
+    assert ct != 0
+
+
+def test_searchBFS():
+    bfs = BreadthFirstSearch(apla)
+    _, ct = bfs.search()
```

Comment:
search method from bfs has two cases of return:

- First one:
```python
if self.automated_planner.satisfies(self.automated_planner.problem.goal, current_node.state):
    computation_time = now() - time_start
    self.automated_planner.logger.debug("Search finished at: " + str(timestamp.now()))
    return current_node, computation_time, opened_nodes
```

- Second one:
```python
return None, computation_time, opened_nodes
```

So it does unpack 3 values everytime right ?